src/components/App/index.tsx

```diff
@@ -27,48 +28,60 @@ export default function App(): React.ReactElement {
   // Add the current theme as a class on the body element
   useBodyClass(themeContextValue[0]);
 
+  const date = new Date();
+  const websiteDown =
+    date.getFullYear() === 2023 &&
+    date.getMonth() === 2 &&
+    (date.getDate() === 15 || date.getDate() === 16);
+
   return (
     <ThemeContext.Provider value={themeContextValue}>
       <AppCSSRoot>
         <TooltipProvider>
-          <ErrorBoundary
-            fallback={(error, errorInfo): React.ReactElement => (
-              <AppSkeleton>
-                <SkeletonContent>
-                  <ErrorHeader />
-                  <ErrorDisplay
-                    errorDetails={
-                      <ReactErrorDetails error={error} errorInfo={errorInfo} />
-                    }
-                  >
-                    <div>
-                      There was en error somewhere in the core application logic
-                      and it can&apos;t continue.
-                    </div>
-                    <div>
-                      Try refreshing the page to see if it fixes the issue.
-                    </div>
-                  </ErrorDisplay>
-                </SkeletonContent>
-              </AppSkeleton>
-            )}
-          >
-            <AppNavigation>
-              {/* AppDataLoader is in charge of ensuring that there are valid values
-                for the Terms and Term contexts before rendering its children.
-                If any data is still loading,
-                then it displays an "app skeleton" with a spinner.
-                If there was an error while loading
-                then it displays an error screen. */}
-              <AppDataLoader>
-                <AppContent />
-              </AppDataLoader>
-            </AppNavigation>
-            <Feedback />
+          {!websiteDown && (
```

Comment:
NIT: Could use a ternary operator here something like `websiteDown ? <content_if_website_down> : <content_if_website_is_not_down`

Reply:
It's on me lol. The workflow for preview is added in `bog-changes-s23` branch (and any branch that checks out from it) only. `main` doesn't have it. I deployed the migration modal the other day to `main` so all preview sites got deleted